syntax = "proto3";



package IRacingTools.Models;

import "FileInfo.proto";
import "TrackLayoutMetadata.proto";

message TelemetryDataFile {
  enum Status {
    STATUS_CREATED = 0;
    STATUS_AVAILABLE = 5;
    STATUS_INVALID = 10;
    STATUS_ERROR = 20;
  }

  string id = 1;
  string alias = 2;

  Status status = 3;

  /**
   * Track Layout ID
   */
  TrackLayoutMetadata track_layout_metadata = 5;

  FileInfo file_info = 10;

  string error_details = 20;
  


};


